docs/process_pages.py

```diff
@@ -258,6 +318,10 @@ def CreateNavigatonBar(root_path: str, max_levels: int, default_header_dict: dic
                         file_output.write("entries:\n")
                         file_output.writelines(list_of_strings)
 
+    if is_locally_built:
+        from KratosMultiphysics.testing.utilities import GetPython3Command
```

Comment:
```suggestion
```

unused

Reply:
It is used within the method, I did the import their so, specially in the CI, you don't need to have Kratos environment to depoly the pages.